app/views/users/index.html.erb

```diff
@@ -1,2 +1,26 @@
-<h1>Users Index Page</h1>
-<p>Here is a list of users:</p>
+<% content_for :head do %>
+  <%= stylesheet_link_tag 'userstyle', media: 'all' %>
+<% end %>
+
+<h1>Here is a list of users:</h1>
+<ul>
+  <% @users.each do |user| %>
+  <div class ="userphotoName">
+    <div class="photo"> 
+      <% if user.photo.present? %>
+        <%= image_tag user.photo, class: "user-photo" %>
+      <% else %>
+        <%= image_tag "default_avatar.png", alt: "Default Avatar", class: "user-photo" %>
+      <% end %>
+    </div>
```

Comment:
- [ ] Well done implementing the required changes but we need to match the wireframe for a better UI, please 🙏🏽 adjust the photo size 👇🏽

| Wireframe | Your design |
|---|---|
| ![image](https://github.com/massazoum/Rails_App/assets/58642949/307b0d92-f3dc-46be-b12a-e2fcbc3504b0) | ![image](https://github.com/massazoum/Rails_App/assets/58642949/3e1a41e4-494c-4179-bd38-e65fecc352a8) |